buildlib/tools/perf-common.yml

```diff
@@ -0,0 +1,43 @@
+parameters:
+  workDir: ''
+  displayName: ''
+  
+steps:
+  - bash: |
+      set -eE -o pipefail
+      SHA_After=$(echo $(Build.SourceVersionMessage) | awk '{print $2}')
+      SHA_Before=$(echo $(Build.SourceVersionMessage) | awk '{print $4}')
+
+      module=""
+      perfxParams=""
+
+      if [[ "${{ parameters.displayName }}" == "Build-UCX" ]]; then
```

Comment:
Minor: Use switch-case expression instead of if-else

Reply:
Done